Authentication/SignUp/View/SignupView.swift

```diff
@@ -33,96 +34,120 @@ public extension SignupViewType {
 /** Default signup view. */
 internal final class SignupView: UIView, SignupViewType, NibLoadable {
     
+    // - Warning: This delegate must be set before calling the `render` function.
```

Comment:
(I'm adding the why to the comment)
In this case the delegate is an unwrapped optional, so if it doesn't have a value, it will crash when calling the render function, or when setting any of the SignUpViewType protocol setters.
We made this like this because the controller is internal and so we know that this will be set before calling anything else, but I added the comment for anyone to know i they have to touch this.
The thing is that if we make it optional and check if it's `.none` or not everytime, it will be a great amount of code to add and I would leave it for afterwards, if maybe we decide that the controller may be replaced or something. Another approach would be to make `termsAndServicesURL` an optional so we can have a default delegate to back us up as first setting.
But I would leave that decision for later on. If you think so I can add a card for this.